staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/types.go

```diff
@@ -34,13 +34,13 @@ const (
 
 // CustomResourceDefinitionSpec describes how a user wants their resource to appear
 type CustomResourceDefinitionSpec struct {
-	// Group is the group this resource belongs in
+	// `group` is the group this resource belongs in
```

Comment:
these should be unquoted and match the json tag

Reply:
Come again, all the names should be unquoted?

Reply:
yes, the godoc should start with the json tag, unquoted